action/protocol/staking/vote_reviser.go

```diff
@@ -37,7 +39,32 @@ func (vr *VoteReviser) Revise(csm CandidateStateManager, height uint64) error {
 		}
 		vr.storeToCache(height, cands)
 	}
-	return vr.flush(height, csm)
+	return nil
```

Comment:
the logic of updating candidate name/operatoraddr/rewardaddr could be put here.

Reply:
fixed